opennms-webapp/src/main/webapp/WEB-INF/jsp/graph/results.jsp

```diff
@@ -198,6 +199,23 @@
             </c:if>
         </h3>
      </div> <!-- panel-heading -->
+       <c:if test="${fn:contains(resultSet.resource.resourceType.label, 'SNMP Interface')}">
+         <c:if test="${fn:contains(resultSet.resource.label,'(*)') != true}">
+           <c:forEach var="attribute" items="${resultSet.resource.attributes}">
+             <c:if test="${fn:contains(attribute.name, 'ifIndex')}">
+               <c:set var="ifIndex" value="${attribute.value}"/>
+             </c:if>
+             <c:if test="${fn:contains(attribute.name, 'nodeId')}">
+               <c:set var="nodeId" value="${attribute.value}"/>
+             </c:if>
+           </c:forEach>
+           <div ng-app="onms-ksc" ng-controller="checkFlowsCtrl" ng-init="getFlowCount(${nodeId}, ${ifIndex}, ${results.start.time}, ${results.end.time})">
```

Comment:
Right now we end up triggering a REST call, which in turns make a call to Elasticsearch every time a user loads  a page with SNMP interfaces. We should find a way to avoid the queries to Elasticsearch if 1) the system is not processing flows or 2) we already know there are flows available.

If this is not feasible, perhaps would could just make it possible to enable/disable the icon i.e. we require some URL to be configured before we attempt to show the icon.